buoy_gazebo/src/MooringForce/MooringForce.cpp

```diff
@@ -254,34 +299,19 @@ void MooringForce::PreUpdate(
   // Improves solution stability dramatically.
   catenarySolver.useExternalScaling = true;
 
-  // Initial guess B = L - V - b
-  double b = 0.0;
-
-  // Catenary Scaling factor
-  double c = 0.0;
-
-  int solverInfo;
-  while ((c <= 1e-5) && (this->dataPtr->L - this->dataPtr->V - b > 0.0)) {
-    // Increment b, to iterate on B
-    b += 1.0;
+  // Initial estimate for B (upper bound).
+  auto BMax = [](double V, double H, double L) -> double
+  {
+    return (L*L - (V*V + H*H)) / (2 * (L - H));
```

Comment:
Consistent spacing around operators
```suggestion
    return (L * L - (V * V + H * H)) / (2 * (L - H));
```